### ISSUE TITLE
Request a VSA design, dev intent, or spot check review for "View Payment History"

### ISSUE THREAD

**Issue by jason-gcio** [department-of-veterans-affairs/va.gov-team#17001]
## Instructions

1. Click or press the gear icon next to the "Labels" heading on the right. Search for your team in the labels list. If your team is not listed in the Labels menu, please leave a comment after you have created this issue. The Product Support team will create a new label for your team.
2. Text inside `<-- comment -->` blocks will not appear in your issue ticket. These comments offer guidance on what information you should provide for each section.

### For reference

**VSA Development Review** - [https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/accessibility/vsa-accessibility-review-process.md#development-review](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/accessibility/vsa-accessibility-review-process.md#development-review)

## Team Name

eBenefits
## Product/Feature Name

View Payments

## Product description

This feature provides a tabular list of payment history, including returned payments.

## Review stage

- [ ] **Design collaboration review** (Review design to discuss accessibility and inclusive design) - This may happen as design evolves.

- [ ] **Development intent review** (Review design to discuss approach to take with development, identify what components and semantic HTML would be appropriate, identify any accessibility issues and how to address) - This would happen as designs are passed to development, before development begins.

- [x] **Spot check review** (Review and follow the steps in the [508 Checklist](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-checklist.md) first.) This happens well before launch, so that the 508 office can review and any issues addressed *before* launch.

- [x] **Other:** Please indicate what you're requesting
We just need to ensure that the is feature is behaving as expected on mobile views.

## Point of contact

Jason Wolf
Jim Adams

## Preferred times for dev intent checkpoint, if applicable

Anytime, please feel free to slack @ us

## Staging URL and credentials, if applicable

https://staging.va.gov/va-payment-history/payments/
user 228
